### PR TITLE
Add better error for VFR check

### PR DESCRIPTION
**Pull request by awolant**
## Category:

**Refactoring**

## Description:

Extended error message for VFR videos in video reader to include the file path.

### Tests:

- [ ] Existing tests apply
- [ ] New tests added
  - [ ] Python tests
  - [ ] GTests
  - [ ] Benchmark
  - [ ] Other
- [ ] N/A

### Documentation
- [ ] Existing documentation applies
- [ ] Documentation updated
  - [ ] Docstring
  - [ ] Doxygen
  - [ ] RST
  - [ ] Jupyter
  - [ ] Other
- [x] N/A

#### Requirements
- [ ] Implements new requirements
- [ ] Affects existing requirements
- [x] N/A

**REQ IDs**: N/A

**JIRA TASK**: N/A

### CODE REVIEW

**Review by JanuszL — APPROVED** [review]

File: dali/operators/reader/loader/video_loader.cc [hunk 1]
```diff
@@ -690,8 +690,9 @@ void VideoLoader::read_file() {
       break;
     } else if (dec_status == VidReqStatus::REQ_ERROR) {
       LOG_LINE << "Request failed" << std::endl;
-      DALI_FAIL("Detected variable frame rate video. The decoder returned frame that is past "
-                "the expected one");
+      DALI_FAIL("The decoder returned frame that is past the expected one."
```
Comment (JanuszL) [inline dali/operators/reader/loader/video_loader.cc:693]: I wonder if it make sense to use `make_string`.

Comment (awolant) [reply dali/operators/reader/loader/video_loader.cc]: Done

**Comment by awolant** [comment]
image

**Comment by awolant** [comment]
image

**Review by JanuszL — APPROVED** [review]

**Comment by awolant** [comment]
!build

File: dali/operators/reader/loader/video_loader.cc [hunk 2]
```diff
@@ -690,8 +690,9 @@ void VideoLoader::read_file() {
       break;
     } else if (dec_status == VidReqStatus::REQ_ERROR) {
       LOG_LINE << "Request failed" << std::endl;
-      DALI_FAIL("Detected variable frame rate video. The decoder returned frame that is past "
-                "the expected one");
+      DALI_FAIL(make_string("The decoder returned frame that is past the expected one. ",
+                "Most likely cause is the variable frame rate video. ",
+                "Filename: ", req.filename));
```
Comment (mzient) [inline dali/operators/reader/loader/video_loader.cc:695]:
```suggestion
      DALI_FAIL(make_string("The decoder returned a frame that is past the expected one. ",
                "The most likely cause is variable frame rate video. ",
                "Filename: ", req.filename));
```

Comment (awolant) [reply dali/operators/reader/loader/video_loader.cc]: Done

**Review by mzient — APPROVED** [review]

**Comment by awolant** [comment]
!build

**Pull request merged**